src/azure-cli-core/azure/cli/core/_profile.py

```diff
@@ -521,15 +521,15 @@ def _try_parse_msi_account_name(account):
                 return parts[0], (None if len(parts) <= 1 else parts[1])
         return None, None
 
-    def get_login_credentials(self, resource=None, subscription_id=None, aux_subscriptions=None):
+    def get_login_credentials(self, resource=None, subscription_id=None, aux_subscriptions=None, aux_tenants=None):
```

Comment:
just to confirm, this change will not only all cli modules right?  need change log